models/models.js

```diff
@@ -0,0 +1,17 @@
+const db = require('../db/connection')
+
+const fetchAllTopics = () => {
+    console.log("in model")
+
+    let queryString = 'SELECT * FROM topics'
+
+    return db.query(queryString)
+    .then((body) => {
+        if (body.rows.length === 0){
+            return Promise.reject({status: 204, msg:'No data here yet'})
```

Comment:
I don't think a 204 really works here, a 204 is more meant to signify that an operation has succeeded and you don't need to do anything else on the page. Consider something like editing and saving a file in Google Docs, we need to know that the file saved correctly but we don't need any data returned really.

A 204 also has no content by default which is why your test failed earlier and you commented it out. In this scenario I would probably just send a 200 and an empty array if we did happen to have no topics for some reason.